edgelet/edgelet-image-cleanup/Cargo.toml

```
@@ -10,10 +10,10 @@ anyhow = "1"
 chrono = "0.4"
 log = "0.4"
 thiserror = "1"
-tokio = "1"
+tokio = { version = "1", features = ["parking_lot"] }
```

```suggestion
tokio = { version = "1", features = ["time"] }
```
This crate only uses `tokio::time::..`. The fact that it was not specified in the features before is surprising.